src/omni/embed.py

```diff
@@ -128,17 +128,18 @@ def build_url(
         return str(url)
 
     def _sign_url(self, url: DashboardEmbedUrl) -> None:
+        # IMPORTANT: base_url must be first and all other attributes must be in alphabetical order.
         blob_items = [
```

Comment:
Is there a way to add a test that makes this more explicit beyond the comment and the signature comparison? Might be nice for the test to enforce order and make it obvious on failure

Reply:
I was actually right the first time and going to close out. The first handful are in a specific order and then the rest of the optionals are in alphabetical. The test catches it based on the signature being correct and matching. Maybe we can get access to a demo account and set up an integration test that actually makes a curl request and validates the url works.